addons/web/static/src/views/calendar/calendar_year/calendar_year_renderer.js

```diff
@@ -35,8 +35,7 @@ export class CalendarYearRenderer extends Component {
 
     get options() {
         return {
-            columnHeaderFormat: (info) =>
-                luxon.DateTime.fromJSDate(info.date.marker).toFormat("EEEEE"),
+            columnHeaderFormat: { weekday: "narrow" },
```

Comment:
As the luxon plugin is activated, use luxon's formatting:  `columnHeaderFormat: "EEEEE",`